documentation/docs/getting-started/decentralized_identifiers/resolve.md

```diff
@@ -5,5 +5,47 @@ description: Explain how resolving works including arguments.
 image: /img/Identity_icon.png
 keywords:
 - Resolve
+- DID
 ---
-TODO: Explain how resolving works including arguments.
\ No newline at end of file
+
+In the previous section we created a decentralized identity and published the associated DID Document to the Tangle; in this section we'll show you how to look up a DID document.
+
+:::info Definition
+
+_DID resolution_ is the process of obtaining a DID document for a given DID.
+
+:::
+
+### Resolving an Identity using the Account
+
+Continuing from the previous example, we'll use the `did` associated with the identity we created to retrieve the published DID document.
+
+```rust
+// Retrieve the DID from the newly created Identity state.
+let did: &IotaDID = snapshot.identity().try_did()?;
+
+// Fetch the DID Document from the Tangle.
+let resolved_document: IotaDocument = account.resolve_identity(did).await?;
+
+println!("[Example] Tangle Document = {:#?}", resolved_document);
+```
+
+In simplified terms, resolution works by querying the Tangle for DID Document messages and returning the most recent valid message that matches the specified DID. In practice, the operation is more complex and is explained in full in the [IOTA DID Method specification](../../specs/iota_did_method_spec#read).
```

Comment:
> the Tangle
Technically would be more accurate as "a Tangle" as the mainnet, devnet, private networks all are separate Tangles that can be used.

Reply:
That's true, but the docs use "the Tangle" throughout so I figured I'd be consistent. Maybe we should fix the wording throughout, or maybe it's understood that "the" in "the Tangle" refers to "the Tangle for which the client is connected to". wdyt?